Examples/Image/Classification/AlexNet/Read_labelclsloc.py

```diff
@@ -0,0 +1,23 @@
+import os
```

Comment:
Minor point about this filename. Can we use something more in convention to other files in CNTK, such as ReadLabels_ClassLoc_ILSVRC.py?